src/main/java/org/spongepowered/api/item/recipe/SmeltingRecipe.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.item.recipe;
+
+import com.google.common.collect.ImmutableList;
+import org.spongepowered.api.Sponge;
+import org.spongepowered.api.item.inventory.ItemStack;
+import org.spongepowered.api.util.ResettableBuilder;
+
+import java.util.List;
+
+public interface SmeltingRecipe extends Recipe {
+
+    /**
+     * Creates a new {@link Builder} to build a {@link SmeltingRecipe}.
+     *
+     * @return The new builder
+     */
+    static Builder builder() {
+        return Sponge.getRegistry().createBuilder(Builder.class);
+    }
+
+    /**
+     * Gets the experience given when this recipe is made.
+     *
+     * @return the experience given when this recipe is made
+     */
+    float getExperience();
+
+    /**
+     * Gets the item needed to make this recipe.
+     *
+     * @return item needed to make this recipe
+     */
+    ItemStack getIngredient();
+
+    /**
+     * Gets the result of this {@link Recipe}.
+     *
+     * @return the result of this {@link Recipe}
+     */
+    ItemStack getResult();
+
+    default List<ItemStack> getResults() {
+        return ImmutableList.of(getResult());
+    }
+
+    interface Builder extends ResettableBuilder<SmeltingRecipe, Builder> {
+        /**
+         * Sets the ingredient for the requirements of this {@link SmeltingRecipe}.
+         *
+         * @param ingredient The ingredient
+         * @return fluent interface
```

Comment:
Oh, I just copied that part of the comment from the other recipe builders, I'll change them too while I'm at it